addons/self-referencing/addon.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Self referencing",
```

Comment:
This needs to be more specific -- something like "self-referencing blocks" or "sprite self-referencing".